tools/ncf.py

```diff
@@ -577,6 +577,14 @@ def get_all_techniques_metadata(include_methods_calls = True, alt_path = ''):
   return { "data": { "techniques" : all_metadata }, "errors": format_errors(errors), "warnings": warnings }
 
 
+def get_agents_support(method, content):
+  agents = []
+  if os.path.exists("/var/rudder/configuration-repository/dsc/ncf/30_generic_methods/" + method + ".ps1"):
+    agents.append("dsc")
+  if not re.search(r'\n\s*bundle\s+agent\s+'+method+r'\b.*?\{\s*\}', content, re.DOTALL): # this matches an empty bundle content
```

Comment:
The `.cf` version of dsc-only methods seems not to be empty but contains a log (https://github.com/Normation/rudder-agent-windows/blob/master/plugin/ncf/30_generic_methods/service_status.cf for example)?